### PR TITLE
Christian/wanting jwt

### PR DESCRIPTION
**Pull request by cmagorr1**
JWT authentication changes

### CODE REVIEW

File: server/.env
Comment (mikebasta) [inline server/.env:1]: This one should be deleted so it doesn't get pushed to the repo

File: server/models/mongooseModels.js
Comment (mikebasta) [inline server/models/mongooseModels.js:1]: line 6 - current code should be used

File: server/models/pgModels.js
Comment (mikebasta) [inline server/models/pgModels.js:1]: process.env should be used

**Review by rjsandman — APPROVED** [review]
Looks good.

**Review by mikebasta — CHANGES_REQUESTED** [review]

**Review by mikebasta — APPROVED** [review]
Nice work!

**Pull request merged**